src/ripple/app/paths/PathRequest.cpp

```diff
@@ -275,7 +276,7 @@ Json::Value PathRequest::doCreate (
         }
     }
 
-    return status;
+    return { valid, status };
```

Comment:
I haven't examined the assembly, but all our compilers are good enough that they should give us close-to-RVO semantics: move `status` into the pair, and RVO the pair.